backend/apps/volontulo/views/auth.py

```diff
@@ -193,27 +193,3 @@ def activate(request, uuid):
             """Brak użytkownika spełniającego wymagane kryteria."""
         )
     return redirect('homepage')
-
-
-def password_reset(request):
-    """View responsible for password reset."""
-    return auth_views.password_reset(
```

Comment:
Remove unused imports of `auth_views` and `FROM_ADDRESS`, please.

Reply:
done!